dataproc/instantiate_inline_workflow_template_test.py

```diff
@@ -13,19 +13,21 @@
 # limitations under the License.
 
 import os
-
-import instantiate_inline_workflow_template
+import subprocess
 
 
 PROJECT_ID = os.environ['GCLOUD_PROJECT']
 REGION = 'us-central1'
 
 
-def test_workflows(capsys):
+def test_workflows():
     # Wrapper function for client library function
-    instantiate_inline_workflow_template.instantiate_inline_workflow_template(
-        PROJECT_ID, REGION
-    )
+    command = [
+        'python', 'instantiate_inline_workflow_template.py',
+        PROJECT_ID,
+        REGION,
+    ]
+
+    out = subprocess.check_output(command).decode("utf-8")
```

Comment:
Please don't spawn a process unless it's extremely necessary.
Options:
1) just keep the current test
2) have`main(argv=None)` and test `main(my_argv)`

```python
def main(argv=sys.argv):
    # use argv in your code....

if __name__ == "__main__":
    main()
```

Reply:
Done. Kept original tests.